components/eam/src/chemistry/utils/modal_aero_calcsize.F90

```diff
@@ -674,21 +666,13 @@ subroutine modal_aero_calcsize_sub(state, deltat, pbuf, ptend, do_adjust_in, &
    endif
 
    !if(present(caller) .and. masterproc) write(iulog,*)'modal_aero_calcsize_sub has been called by ', trim(caller)
-
-   pdel     => state%pdel !Only required if update_mmr = .true.
-   state_q  => state%q    !BSINGH - it is okay to use state_q for num mmr but not for specie mmr (as diagnostic call may miss some species)
-
    !----------------------------------------------------------------------------
```

Comment:
Can the above comment also be removed?

Reply:
These comments were already removed. I may be missing something :confused:

Reply:
I meant the line 668, " !if(present(caller) .and. masterproc) write(iulog,*)'modal_aero_calcsize_sub has been called by ', trim(caller)"  (I might have inserted this comment in the wrong place, sorry about that.)

Reply:
oh, okay...I have removed it now. I didn't pay much attention to calcsize module as we have already refactored and ported it. I did the minimal so that I could remove state and pbuf from all of the optics.

Reply:
I have removed this code now and pushed the latest changes. Currently running the tests, and if they all succeed, I'll proceed with the merge.